docs/getting_started/serial_communication.md

```diff
@@ -1,20 +1,20 @@
 # Board to PC communication over USB
 
-The mbed microcontroller on your board can communicate with a host PC over the same USB cable that is used for programming.
+The mbed microcontroller on your board can communicate with a host PC over the same USB cable that you use for programming.
 
-<span class="tips">If you're working on Windows, you might need to [install a serial driver](what_need.md#windows-serial-driver).</span>
+<span class="tips">If you're working on Windows ealier than Windows 10, you might need to [install a serial driver](what_need.md#windows-serial-driver).</span>
```

Comment:
Now I'm wondering if we want to say "older than Windows 10". 
Decisiveness is not my middle name.